src/View.Uno/Extensions/PasswordBox/PasswordEnterCommand.Xamarin.cs

```diff
@@ -6,12 +6,7 @@
 using Uno.Extensions;
```

Comment:
Add if not WINUI on top

Reply:
This file is in the Uno project, there's no need target related to WinUI